ipaserver/install/adtrustinstance.py

```diff
@@ -482,6 +482,18 @@ def __add_cldap_module(self):
         """
         self.__add_plugin_conf('CLDAP', 'ipa_cldap', 'ipa-cldap-conf.ldif')
 
+    def __allow_trust_attr_mod(self):
+        """
+        aci to allow users to manage own trust attributes
+        """
+        user_aci = paths.UPDATES_DIR + "75-user-trust-attributes.update"
```

Comment:
I think this is neither necessary nor won't this code work properly.

* ``ipa-server-upgrade`` automatically applies all update files. It should not be necessary to run the update manually.
* update files aren't LDIF files. They look a bit like LDIF, but they have a different syntax. ldapmodify can only process LDIF files. Update files are handled by ``ipaserver.install.ldapupdate.LDAPUpdate``.

Reply:
Got it.